storage/remote/client.go

```diff
@@ -151,7 +151,17 @@ func (c *Client) Read(ctx context.Context, query *prompb.Query) (*prompb.QueryRe
 	httpReq.Header.Set("User-Agent", userAgent)
 	httpReq.Header.Set("X-Prometheus-Remote-Read-Version", "0.1.0")
 
-	if span := opentracing.SpanFromContext(ctx); span != nil {
+	var span opentracing.Span
```

Comment:
can be moved inside the following if()

Reply:
Not if I want to close it just after the http request is done